ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/privacy/EeaSendRawTransaction.java

```diff
@@ -116,8 +115,8 @@ private Transaction handlePrivateTransaction(final PrivateTransaction privateTra
     try {
       return privateTransactionHandler.handle(
           privateTransaction, () -> getNonce(privateTransaction.getSender()));
-    } catch (final IOException e) {
-      throw new InvalidJsonRpcRequestException("Unable to handle private transaction", e);
+    } catch (final Exception e) {
+      throw new InvalidJsonRpcRequestException(e.getMessage());
```

Comment:
this was changed in https://github.com/PegaSysEng/pantheon/pull/1449

Reply:
Will update once the PR is merged

Reply:
Rethrow should include the whole traceback, or add a comment if there is a reason we don't want to do that.